src/components/Articles/ResolutionDates.vue

```diff
@@ -9,7 +9,7 @@
         </label>
       </v-col>
 
-      <v-col :cols="isEditMode ? '7' : '9'">
+      <v-col :cols="isEditMode ? '7' : '8'">
```

Comment:
The columns now add up to 3+8=11 instead of 12, but this gives right side whitespace to match the comps.